modules/processing/snowflake/sql/ST_VORONOIPOLYGONS.sql

```diff
@@ -0,0 +1,17 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE SECURE FUNCTION @@SF_PREFIX@@processing.ST_VORONOIPOLYGONS
+(points ARRAY, bbox ARRAY)
+RETURNS ARRAY
+AS $$(
+    @@SF_PREFIX@@processing._VORONOIHELPER(points, bbox, 'poly')
+)$$;
+
+CREATE OR REPLACE SECURE FUNCTION @@SF_PREFIX@@processing.ST_VORONOIPOLYGONS
+(points ARRAY)
+RETURNS ARRAY
+AS $$(
+    @@SF_PREFIX@@processing._VORONOIHELPER(points, null, 'poly')
```

Comment:
Instead of passing null we should pass [-180,-85,180,-85]. 
In snowflake we are the possibility of passing null to use default params. And on your current code:  `sfcarto.processing.ST_VORONOIPOLYGONS(an array of points, null) would work

Reply:
Ok!